openc3/lib/openc3/accessors/binary_accessor.rb

```diff
@@ -115,6 +115,149 @@ def self.raise_buffer_error(read_write, buffer, data_type, given_bit_offset, giv
     # Valid endianess
     ENDIANNESS = [:BIG_ENDIAN, :LITTLE_ENDIAN]
 
+    def handle_read_variable_bit_size(item, _buffer)
+      length_value = @packet.read(item.variable_bit_size['length_item_name'], :CONVERTED)
+      if item.array_size
+        item.array_size = (length_value * item.variable_bit_size['length_bits_per_count']) + item.variable_bit_size['length_value_bit_offset']
+      else
+        if item.data_type == :INT or item.data_type == :UINT
+          # QUIC encoding is currently assumed for individual variable sized integers
+          case length_value
+          when 0
+            item.bit_size = 6
+          when 1
+            item.bit_size = 14
+          when 2
+            item.bit_size = 30
+          else
+            item.bit_size = 62
+          end
```

Comment:
Why are they all 2 bits short?

Reply:
Its the QUIC variable size integer format.  See: https://datatracker.ietf.org/doc/html/rfc9000#name-variable-length-integer-enc

Reply:
That's above and beyond ... I thought you were just going to do strings, lol